client/my-sites/site-settings/settings-section-header.tsx

```diff
@@ -1,40 +1,38 @@
 import { Button } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import PropTypes from 'prop-types';
+import { FC, PropsWithChildren, ReactNode } from 'react';
 import SectionHeader from 'calypso/components/section-header';
 
-const SettingsSectionHeader = ( {
+interface SettingsSectionHeaderProps extends PropsWithChildren {
+	title: string | ReactNode;
+	id?: string;
+	isSaving?: boolean;
+	showButton?: boolean;
+	disabled?: boolean;
+	onButtonClick?: () => void;
+}
+
+const SettingsSectionHeader: FC< SettingsSectionHeaderProps > = ( {
```

Comment:
FC is not necessary and largely in disuse these days. Components are simply functions, so let's just type the props (see suggestion below).

```suggestion
function SettingsSectionHeader( {
```

Reply:
Similar to `PropsWithChildren` seems like a matter of taste:

fbhepr%2Sfrnepu%3Scebwrpg%3Qpnylcfb%26shyy%3QSP%253P%26qrsf%3Q%26ersf%3Q%26cngu%3Q%26uvfg%3Q%26glcr%3Q%26keq%3Q%26aa%3Q1-og

Reply:
The `FC` helper is indeed a bit dated nowadays, the contemporary taste is to just write plain functions. Because modern functional components are nothing more. The `FC` type also defines types for `propTypes`, `defaultProps` or `displayName`, but new code rarely uses these, especially TypeScript code.

Reply:
Thanks for the explanation folks, it should be fixed already 😄